src/test/EigenPod.t.sol

```diff
@@ -290,6 +290,37 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         );
     }
 
+    function testWithdrawBeforeRestakingWithTimestampChecks() public {
```

Comment:
seems to me that this can be combines with the `testWithdrawBeforeRestaking` test

Reply:
what did you decide on doing here?